continuous_learning_system.py

```
@@ -0,0 +1,605 @@
+                    SET end_time = ?, status = ?
+                    WHERE session_id = ?
+                """, (
+                    datetime.now().isoformat(),
+                    'completed',
+                    session_id
+                ))
+                conn.commit()
+                
+            # 学習統計ログ
+            duration = datetime.now() - self.start_time
+            self.logger.info(f"Learning session completed:")
```

Ruff (0.12.2)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)